### DISCUSSION TITLE
iPad/iOS support would be cool you know

### DISCUSSION THREAD

**Discussion by aj-arts** [pystardust/ani-cli#195] [Ideas] [answered]
is there any way this can work on iPad and the history file which keeps track of your episode no will get synced across ur iPad and Mac and we can resume it while seamlessly switching back and forth between Mac on iPad?

**Comment by port19x** [comment, marked as answer]
You will definitely have to do syncing on your own. You just have to sync a file for that.

iOS support would be great. Consider opening an issue requesting this feature.
Don't count on it being supported anytime soon, but in a year things may look different

**Comment by ghost** [comment]
you'd def need a jailbreak for this
